files/src/main/scalajvm/sttp/tapir/files/model.scala

```diff
@@ -15,8 +15,9 @@ case class StaticInput(
     range: Option[Range],
     acceptEncoding: Option[String]
 ) {
-
-  def acceptGzip: Boolean = acceptEncoding.contains("gzip")
+  val acceptedEncodings: List[String] =
```

Comment:
This solution is fine, but maybe it would be better to move this parsing step into the header description, as it's done e.g. here: https://github.com/softwaremill/tapir/blob/7a9e8872a7e3faaf2f7d94bb7e4ab01beb72c80b/files/src/main/scalajvm/sttp/tapir/files/TapirStaticContentEndpoints.scala#L31